packages/foam-vscode/src/core/model/workspace.test.ts

```diff
@@ -184,6 +199,44 @@ describe('Identifier computation', () => {
       'to/page-a#section name'
     );
   });
+
```

Comment:
Piggyback: here I just moved the `getShortestIdentifier` tests to the `'Identifier computation'` section. In my previous refactor I moved `getShortestIdentifier` from `utils/core.ts` to `model/workspace.ts` alongside its tests, but didn't realize that there already was a section for them.